templates/kotlin/src/main/kotlin/io/appwrite/Role.kt.twig

```diff
@@ -1,29 +1,55 @@
 package {{ sdk.namespace | caseDot }}
 
-class Role {
-    companion object {
-        fun any(): String = "any"
-
-        fun user(id: String, status: String = ""): String = if(status.isEmpty()) {
-            "user:$id"
-        } else {
-            "user:$id/$status"
-        }
-
-        fun users(status: String = ""): String = if(status.isEmpty()) {
-            "users"
-        } else {
-            "users/$status"
-        }
-
-        fun guests(): String = "guests"
-
-        fun team(id: String, role: String = ""): String = if(role.isEmpty()) {
-            "team:$id"
-        } else {
-            "team:$id/$role"
-        }
-
-        fun member(id: String): String = "member:$id"
-    }
+/**
+ * Helper class for role selectors
+ */
+object Role {
+    const val any = "any"
+
+    const val guests = "guests"
+
+    @Deprecated("Use const val", ReplaceWith("any"))
+    @JvmStatic
+    fun any() = any
+
+    /**
+     * Single user with the [id] (and [status], if provided) is selected
+     * 
+     * @param id user ID to select
+     * @param status user status to select
+     */
+    @JvmStatic
+    @JvmOverloads
+    fun user(id: String, status: String? = null) = "user:$id${status?.let { "/$it" } ?: ""}"
```

Comment:
Let's keep the `status` default as an empty string and keep the method body as an `if/else` condition instead of using `.let`, for all methods in this object

Reply:
`?.let` will essentially inline to an if statement like "value is null? return empty string, else add a leading slash to it". I personally prefer nulls over empty strings so that is up to the dev